packages/web/src/pages/explore-page/components/desktop/TrendingPlaylists.tsx

```diff
@@ -0,0 +1,162 @@
+import { useTrendingPlaylists, UseLineupQueryData } from '@audius/common/api'
+import { exploreMessages as messages } from '@audius/common/messages'
+import { ID, PlaybackSource, Status, UID } from '@audius/common/models'
+import { Flex } from '@audius/harmony'
+import { ClassNames } from '@emotion/react'
+
+import { CollectionTile as DesktopCollectionTile } from 'components/track/desktop/CollectionTile'
+import { CollectionTile as MobileCollectionTile } from 'components/track/mobile/CollectionTile'
+import { TrackTileSize } from 'components/track/types'
+import { useIsMobile } from 'hooks/useIsMobile'
+
+import { Carousel } from './Carousel'
+import {
+  COLLECTION_TILE_HEIGHT,
+  MOBILE_COLLECTION_TILE_HEIGHT,
+  MOBILE_TILE_WIDTH,
+  TILE_WIDTH
+} from './constants'
+
+type TileType = typeof DesktopCollectionTile | typeof MobileCollectionTile
+
+const CollectionTileSkeleton = ({
+  size,
+  Tile
+}: {
+  size: TrackTileSize
+  Tile: TileType
+}) => {
+  return (
+    <ClassNames>
```

Comment:
remind me why we need the ClassNames wrapper?

Reply:
This is the recommended method in the emotion docs if you're using React.